resources/definitions/fdmprinter.def.json

```diff
@@ -6954,6 +6954,203 @@
                     "type": "float",
                     "enabled": "bridge_settings_enabled and bridge_enable_more_layers",
                     "settable_per_mesh": true
+                },
+                "clean_between_layers": 
```

Comment:
In the engine (see my review there) you have made it so that `clean_between_layers` is mutually exclusive with `retract_at_layer_change`. This is fine, but should be handled here; that is, the values both `"enabled"`-fields of those options should be set to false if the other is true.

Reply:
Logic in the engine was left as it is. But description for this setting was clarified to make it more obvious for user how this will interact with `retract_at_layer_change` behavior.